File: Source code/src/main/java/bom/Order.java
package bom;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@AllArgsConstructor
@NoArgsConstructor
public class Order {
	
	private int id;
	
	private OderStatus status;
	
	private int total;
	
	private User user;
	
	private String comment;
}

File: Source code/classes/production/paper-dashboard-angular-master/shared/models/order.ts
import {LoginUser} from './login-user';

export interface Order {
  id: number;
  status: OrderStatus;
  total: number;
  user: LoginUser;
  comment: string;
}

enum OrderStatus {
  IN_CREATED = 'IN_CREATED',
  OUT_CREATED = 'OUT_CREATED',
  RUNNING = 'RUNNING',
  DONE = 'DONE',
  FAILURE = 'FAILURE',
}
